includes/class-wc-install.php

```diff
@@ -806,6 +806,20 @@ private static function get_schema() {
   orders_net_total double DEFAULT 0 NOT NULL,
   PRIMARY KEY (start_time)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}wc_order_product_lookup (
+  order_id BIGINT UNSIGNED NOT NULL,
+  product_id BIGINT UNSIGNED NOT NULL,
+  order_item_id BIGINT UNSIGNED NOT NULL,
+  customer_id BIGINT UNSIGNED NULL,
+  date_completed datetime NOT NULL,
```

Comment:
I think this should probably be using date_created instead of date_completed, as there will be `processing` and `on-hold` orders in here also.